tests/benchmarks/exercism_practice_python.py

```diff
@@ -0,0 +1,174 @@
+import concurrent.futures
+import os
+import subprocess
+import sys
+import threading
+from textwrap import dedent
+
+import pytest
+from git import Repo
+
+from mentat.app import run
+from mentat.user_input_manager import UserInputManager, UserQuitInterrupt
+
+threadLocal = threading.local()
+
+
+def exercise_passed():
+    try:
+        with open(threadLocal.test_output_file, "r") as f:
+            lines = f.readlines()
+            return "failed" not in lines[-1] and "passed" in lines[-1]
+    except FileNotFoundError:
+        return False
+
+
+def get_error_message():
+    with open(threadLocal.test_output_file, "r") as f:
+        lines = f.readlines()
+        lines = lines[-30:]
```

Comment:
I saw Aider sends "the first 50 lines of test error". It makes more sense to me to send the last lines like you're doing here. Is 30 the most that fits in context sometimes?